vunit/ui.py

```diff
@@ -870,6 +877,20 @@ def _main(self, post_run):
         all_ok = self._main_run(post_run)
         return all_ok
 
+    def _add_ext_pkg(self):

```

Comment:
I think this needs a better name than "ext_pkg"

Reply:
What is your suggestion? I have considered `external_memory_pkg`, `extmem_pkg`, `external_pkg`, `ext_pkg`, `vhpi_pkg`, etc.

Reply:
The purpose of the package is to select the backend/implementation of the byte array.
Thus the method could be called: `set_external_byte_array_implementation`

Reply:
I am considering to propose the contribution of 'external' modes for three 'types': `byte_vector` (`string`), `integer_vector` and `memory`. Should we name the method `set_external_array_implementations`? Or do we want to have up to three different methods?

Reply:
The function is named `add_external_arrays` now. It accepts two optional arguments:

- `packages = [False, False, False]`: whether external packages should be loaded. The first item corresponds to `byte_vector`, the second one to `integer_vector` and the third one to `memory`. For now, the first one is used only.
- `implementations = [[], [], []]`: if the previous is True, optionally provide the list of VHDL files that implement the packages. For now, the first list is used only.